keyboards/converter/usb_usb/keymaps/bepo/keymap.c

```diff
@@ -0,0 +1,206 @@
+/*
+	Transform a BÉPO keyboard into a standart AZERTY keyboard based on a iso layoutPO
+	
+	Use only when you cannot install the native BÉPO driver
+	as for instance if you are not allowed to install program / drivers on a business computer
+	Your windows has to be set with a French AZERTY input method (laptop keyboard will stay in azerty for instance)
+	
+	NOTES:
+	* All special code are specific to Windows (ALT+wincode input method)
+	* Many dead letters are not implemented (lack of alt-code for that)

```

Comment:
If you really want to, you can still use the Alt Codes of the Combining Diacritical block such as Alt+0303 for [Combining Tilde](https://www.fileformat.info/info/unicode/char/0303/index.htm). The downside is that it is used in the other direction compared to dead keys (when using dead keys, you first press the dead accent key and then the letter you want the accent on whereas for combining diacritics, you first type the letter and then the combining accent to apply it to the previous letter) on top of requiring pressing backspace twice to delete.

Reply:
Thanks! Even if most unicode alt entries only works in Word and limited applications (up to my tests so far!), I implemented the dead car available on a azerty keyboard (so only 3: ~, ¨, ^), but good hint to try to add some others (not needed for French, but would be closer to a full real bepo keyboard.
Note that the assumption that I cannot change whatever on the OS that requires admin access or process execution (so alternative solutions, with WinCompose or unicode registry change is not valid in my case).

Reply:
On Belgian Azerty, there are more available dead keys: `'`, <code>\`</code>, `^`, `¨`, `~`

Reply:
US Intl for instance was considered, and other available layouts. The "issue" here is that it would affect the laptop integrated keyboard at the same time. When moving to a meeting and so, it is more convenient not to change the Windows regional settings to have the (other) keyboard I know.